doc/architecture/adr/0002-pull-acs-info-out-of-modules.md

```diff
@@ -0,0 +1,62 @@
+# Pull acs-info out of modules
+
+* Status: accepted 
+* Deciders: Gary Crye, Jamie Visker, Josh Gubler, Scott Hutchings
+* Date: 2020-02-20
+
+<!-- Technical Story: [description | ticket/issue URL] optional -->
+
+## Context and Problem Statement
+
+Using acs-info inside our standard and component modules is creating pain. Does it make more sense to expect products to pass the necessary variables in, allowing them to use acs-info if they want?
+
+## Decision Drivers <!-- optional -->
+
+* Terraform's lack of optimization with modules and data sources
+* Usage of modules in non-OIT accounts
+* We want our modules to be simple to use
+* We want development teams to understand the infrastructure of their products.
+
+## Considered Options
+
+* Pull acs-info out of modules
+* Leave acs-info in modules
+
+## Decision Outcome
+
+Chosen option: "Pull acs-info out of modules", because  comes out best (see below).
```

Comment:
because what comes out best?

Reply:
That was language from the MADR template. It basically means, "as we went through the pros/cons listed below, this option was the best."

I'll try and come up with better wording.

Reply:
Done